--- gfax/src/Makefile.am.orig	2005-02-21 20:45:11.000000000 +0100
+++ gfax/src/Makefile.am	2005-05-03 02:01:02.000000000 +0200
@@ -46,14 +46,14 @@
 
 install-data-local: $(TARGET)
 	$(mkinstalldirs) $(DESTDIR)$(pkglibdir)
-	$(mkinstalldirs) -m 777 $(eprefix)/var/spool/gfax
-	$(mkinstalldirs) -m 777 $(eprefix)/var/spool/gfax/doneq
-	$(mkinstalldirs) -m 777 $(eprefix)/var/spool/gfax/recq
+	$(mkinstalldirs) -m 1777 $(DESTDIR)$(eprefix)/var/spool/gfax
+	$(mkinstalldirs) -m 1777 $(DESTDIR)$(eprefix)/var/spool/gfax/doneq
+	$(mkinstalldirs) -m 1777 $(DESTDIR)$(eprefix)/var/spool/gfax/recq
 	$(INSTALL_PROGRAM) $(TARGET) $(DESTDIR)$(pkglibdir)/$(TARGET)
 
 uninstall-local:
 	rm -f $(DESTDIR)$(pkglibdir)/$(TARGET)
-	rm -rf $(eprefix)/var/spool/gfax
+	rm -rf $(DESTDIR)$(eprefix)/var/spool/gfax
 
 EXTRA_DIST =            		\
 	$(srcdir)/Defines.cs.in		\
