--- gfax/gfax.desktop.orig	2005-02-28 23:13:26.000000000 +0100
+++ gfax/gfax.desktop	2005-05-02 23:51:54.000000000 +0200
@@ -1,15 +1,15 @@
 [Desktop Entry]
 Version=1.0
 Encoding=UTF-8
-Name=Gfax Facsimile Sender
-Name[de]=Gfax Faxversand
-Name[en_CA]=Gfax Facsimile Sender
+Name=Gfax
 Comment=Send Facsimiles
 Comment[de]=Faxe versenden
 Comment[en_CA]=Send Facsimiles
+Comment[pl]=Wysyłanie faksów
 Exec=gfax
 Icon=gfax.png
 StartupNotify=true
 Terminal=false
 Type=Application
-Categories=Application;Office;
+Categories=GTK;Office;X-Misc;
+# vi: encoding=utf-8
